wsclient/pom.xml

```diff
@@ -3,7 +3,7 @@
     <groupId>org.hpccsystems</groupId>
     <artifactId>wsclient</artifactId>
     <packaging>jar</packaging>
-    <version>7.0.0-SNAPSHOT</version>
+    <version>7.0.2-SNAPSHOT</version>
```

Comment:
master branch should be set to 7.1.0-snapshot

Reply:
K will do.